File: src/LabOnline.UI/angularApp/app/models/examOrder.ts
import { ExamLine } from './examLine';
export class ExamOrder {
	public isSelected = false;
    public examDate: string;
    public examId: string;
    public patientName: string;
    public patientId: string;
    public processed: string;
    public lineItems: ExamLine[];
}

File: src/LabOnline.Business/Domain/ExamOrder.cs
using System.Collections.Generic;

namespace LabOnline.Business.Domain
{
    public class ExamOrder
    {
        public string ExamDate { get; set; }
        public string ExamId { get; set; }
        public string PatientName { get; set; }
        public string PatientId { get; set; }        
        public List<ExamLine> LineItems { get; set; }
        public override string ToString()
        {
            return ExamId ?? base.ToString();
        }
    }
}
